autoload/maktaba/plugin.vim

```diff
@@ -102,6 +111,30 @@ function! s:SanitizedName(name) abort
 endfunction
 
 
+""
+" Gets a dictionary of {leaf path} for each path found on &rtp that does not
+" correspond to a registered plugin.
```

Comment:
It wasn't clear to me what the returned dictionary would look like. Perhaps reference `maktaba#rtp#LeafDirs()` explicitly? (That is, this function returns the subset of `maktaba#rtp#LeafDirs()` that isn't already registered.)

Could also mention that results are cached so that repeated calls are fast.